marimo/_server/api/virtual_file.py

```diff
@@ -9,12 +9,11 @@
 from marimo import _loggers
 from marimo._runtime.virtual_file import EMPTY_VIRTUAL_FILE
 from marimo._server.api.status import HTTPStatus
-from marimo._server.api.validated_handler import ValidatedHandler
 
 LOGGER = _loggers.marimo_logger()
 
 
-class VirtualFileHandler(ValidatedHandler):
+class VirtualFileHandler(tornado.web.RequestHandler):
```

Comment:
since we use this in `<a href` and internal libraries, we cant control the ability to add headers

Reply:
cc @akshayka

Reply:
oh ok -- that should be fine